test/forms/UkimsNumberFormProviderSpec.scala

```diff
@@ -37,13 +37,6 @@ class UkimsNumberFormProviderSpec extends StringFieldBehaviours {
       stringsWithMaxLength(maxLength)
     )
 
-    behave like fieldWithMaxLength(
```

Comment:
i think we should keep this test inn and change what was making it fail

Reply:
Happy to do that I just figure we are gonna remove the test later today when we do validation anyway